src/commands/validate.js

```diff
@@ -84,16 +85,18 @@ export const handler = (config) => {
   forEachProject(projects, projectNames, (project, projectConfig) => {
     const {
       variations,
+      variationsRoot,
       require: requires,
     } = projectConfig;
+    const variationsPath = variationsRoot ? path.join(variationsRoot, variations) : variations;
     const log = x => console.log(`${chalk.inverse(chalk.blue(`Project “${project}”`))}: ${x}`);
     log(chalk.yellow('validating...'));
 
     if (requires) { requireFiles(requires); }
     // the purpose of the try/catch here is so that when an error is encountered, we can continue showing useful output rather than terminating the process.
     try {
       const exitCode = getOverallErrors({
-        variations: globToFiles(variations),
+        variations: globToFiles(variationsPath),
```

Comment:
globToFiles already takes a `projectRoot` argument; this should do `globToFiles(variations, actualRoot)` instead